2021-11-24 21:40:50,baseAPI(line:118),ERROR|Traceback (most recent call last):
  File "G:\SongQin\Python\Demo\Delivery_System_1108\common\baseAPI.py", line 113, in define_api_assert
    assert result == exp_result
AssertionError

2021-11-24 21:40:50,baseAPI(line:118),ERROR|Traceback (most recent call last):
  File "G:\SongQin\Python\Demo\Delivery_System_1108\common\baseAPI.py", line 113, in define_api_assert
    assert result == exp_result
AssertionError

